myproject/myproject/settings.py

```diff
@@ -0,0 +1,149 @@
+"""
+Django settings for myproject project.
+
+Generated by 'django-admin startproject' using Django 5.0.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-5m!xl#lg1rl0+5mu%xi^ft#^2$uk7mpo0h^hsf+++7efz+ntp9'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = [
+    '10.0.1.3',
+    '127.0.0.1',
+]
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        }, 'file': {
+            'class': 'logging.FileHandler',
+            'filename': 'django.log',
+        },
+    },
+    'loggers': {
+        'django': {
+            'handlers': ['console', 'file'],
+            'level': 'INFO',
+        }, 'random_app': {
+            'handlers': ['console','file'],
+            'level': 'DEBUG',
+            'propagate': True,
+        }, },
+}
+
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'hello_django',
+    'random_app',
+    'about_app',
+    'author',
```

Comment:
А что за приложения такие? 
```suggestion
    #  'hello_django',
    #  'random_app',
    'about_app',
    #  'author',
```
Таких приложений нет. Комментирование тоже не помогает. Из-за их отсутствия сервер не запускается. Разве вы этого не заметили?